### ISSUE TITLE
Automatically ignore domain names when spell checking

### ISSUE THREAD

**Issue by chdsbd** [streetsidesoftware/vscode-spell-checker#2606]
Domain names will frequently have invalid words in them, but that's not something the spell checker should complain about. If the spell checker encounters a domain name, it should skip spell checking.

For example,

```
news.ycombinator.com  // "ycombinator": Unknown word.
```

But the desired behavior would be to ignore this name
